### PR TITLE
DX-2115

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

File: site/docs/numbers/about.mdx [hunk 1]
```diff
@@ -10,6 +10,14 @@ hide_title: true
...
 ---
 
+export const Svg = require('../../static/img/icon-numbers.svg').default;
+
+<Svg className='about-image' /> <br/>
+
 The Bandwidth Dashboard API is where you will purchase, port, provision, and manage all of the telephone numbers in your inventory that you use for calls and messages on the Bandwidth network.
 
-We recommend familiarizing yourself with the [Account Structure](./account/structure) and [Account Setup](./account/setup) guides to ensure everything you will need for your provisioning API calls before beginning testing/development. 
+We recommend familiarizing yourself with the [Account Structure](./account/structure) and [Account Setup](./account/setup) guides to ensure everything you will need for your provisioning API calls before beginning testing/development.
```
Comment (dyeam-bw) [inline site/docs/numbers/about.mdx:19]: > to ensure everything you will need for your provisioning API calls before beginning testing/development.
Maybe I'm being picky. Doesn't read right to me. Suggestion:
> to ensure everything you will need for provisioning API calls is set up before beginning your testing or development.

**Review by dyeam-bw — APPROVED** [review]

File: site/docs/numbers/about.mdx (hunk 1 above)
Comment (ajrice6713) [reply site/docs/numbers/about.mdx]: I like that better 👍

**Comment by hamermike** [comment]
I know you didn't add it in this commit but could you remove the `hide_title` attribute? I like the titles to know where I'm at and add some context. I think either "About" or "Numbers" works here as it's the "Numbers" landing page.

**Comment by ajrice6713** [comment]
@hamermike I've been leaving the titles off of only the About/Title/Landing pages on purpose, i think it looks more like a Title page than having a big 'About' left-aligned on the page

Maybe we can do some nice react-y title on each of the product pages - I added the image centered here but maybe some title text underneath? Thoughts?

**Comment by hamermike** [comment]
@ajrice6713 I think the title would be better above the image where it is on the other pages. This keeps the consistency, follows Docusaurus, and shows where the user is as the first thing when viewing the page. I don't think that needs any sort of customization.

**Review by hamermike — APPROVED** [review]

**Pull request merged**